mbtools/html_to_json.py

```diff
@@ -1,22 +1,48 @@
 import argparse
-import glob
+import os
 import json
 from pathlib import Path
 
 
-def create_json_content(uuid, content, variant="main"):
+def create_json_content(uuid, content):
+
     json_content = {"id": uuid, "content":
-                    [{"variant": variant, "html": content}]}
```

Comment:
Minor: This was a little hard for me to read at first. What do you hink about instead:

```python
    json_content = {
        "id": uuid,
        "content": content
    }
```